types/ioredis/index.d.ts

```diff
@@ -873,11 +873,14 @@ declare namespace IORedis {
 
     type ClusterNode = string | number | NodeConfiguration;
 
-    interface Cluster extends NodeJS.EventEmitter, Commander {
-        new(nodes: ClusterNode[], options?: ClusterOptions): Redis;
+    interface Cluster {
```

Comment:
It seems `Cluster` still need ` extends NodeJS.EventEmitter, Commander`, or no `cluster.get`: https://github.com/luin/ioredis#cluster